lighthouse-core/report/v2/renderer/category-renderer.js

```diff
@@ -225,19 +225,52 @@ class CategoryRenderer {
     return element;
   }
 
+  /**
+   * Find the total number of audits contained within a section.
+   * Accounts for nested subsections like Accessibility.
+   * @param {!Array<!Element>} elements
+   * @return {number}
+   */
+  _getTotalAuditsLength(elements) {
+    // Create a scratch element to append sections to so we can reuse querySelectorAll().
+    const scratch = this._dom.createElement('div');
+    elements.forEach(function(element) {
+      scratch.appendChild(element);
+    });
+    const subAudits = scratch.querySelectorAll('.lh-audit-group .lh-audit');
```

Comment:
can we just query for `.lh-audit` and always return that number?

Reply:
sure! would you like me to keep the failure check that returns `elements.length` if subAudits is 0?

Reply:
querying for `.lh-audit` *should* just work if I'm remember correctly, if they return different results though I'm probably misremembering and you can ignore me :)